.github/workflows/main.yml

```diff
@@ -23,42 +23,14 @@ jobs:
       with:
         node-version: ${{ matrix.node-version }}
 
-    - name: Change Directory
-      run: cd path/to/your/project
+    - name: Clean install
```

Comment:
To respect the commit title convention, please prefix your message with the kind of action you are addressing: Fix/Chore/Refactor/Feature. I guess "Feature: Add GitHub actions workflow"

Reply:
done